Document Fleet UI updates (#4828)
langchain-ai/docs: src/langsmith/fleet/templates.mdx

    Scores applicants and flags the strongest candidates.
  </Card>
  <Card title="Recruiting Outreach" icon="user-search">
    Finds candidates worth reaching out to and drafts your outreach.
  </Card>
  <Card title="Competitor Intelligence Slack Bot" icon="brand-slack">
    Answers Slack questions from a maintained competitor battlecard.
  </Card>
  <Card title="Brand Copywriter" icon="pencil">
    Reviews and drafts blog posts and marketing copy.
  </Card>
  <Card title="Executive Assistant" icon="mail">
    Manages your inbox, calendar, and daily brief.
  </Card>
  <Card title="GTM Agent" icon="target">
    A go-to-market teammate for outbound and research.
  </Card>
  <Card title="On-Call Copilot" icon="bell">
    Turns a noisy page into a triaged ticket and a draft response.
  </Card>
  <Card title="Software Engineer" icon="code">
    Ships code from Slack, Linear, and GitHub in a sandbox.
  </Card>
  <Card title="X Content Manager" icon="brand-x">
    Posts on X in your voice, based on what is actually happening.
  </Card>
  <Card title="Tavily Competitor Research" icon="world-search">
    Produces concise, cited competitor insights.
  </Card>
</CardGroup>

<Info>

> The available templates may change over time. To see the current set, open **Templates** in Fleet or the [templates gallery](https://www.langchain.com/templates).

Reviewer: ```suggestion
The available templates may change over time. For the most up-to-date set, open **Templates** in Fleet or the [templates gallery](https://www.langchain.com/templates).
```

Author: updated
